stretch_navigation/CMakeLists.txt

```diff
@@ -1,204 +1,40 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Do we need a CMakeLists? In Stretch Core, we just use [setup.py](https://github.com/hello-robot/stretch_ros2/blob/galactic/stretch_core/setup.py) and [setup.cfg](https://github.com/hello-robot/stretch_ros2/blob/galactic/stretch_core/setup.cfg)

Reply:
This is a forward-looking change that might be necessary while working with BehaviorTrees.cpp. If users want to experiment with it, I thought this would allow it.